about/index.rst

```diff
@@ -11,6 +11,7 @@ About
    list_of_features
    docs_changelog
    release_policy
+   complying_with_licenses
```

Comment:
Looks like this got mixed up with https://github.com/godotengine/godot-docs/pull/4224